js/presets.js

```diff
@@ -83,11 +83,11 @@ export default {
       const yellowPlate = plates[70] // 70 hue
       const violetPlate = plates[260] // 260 hue
       const greenPlate = plates[130] // 130 hue
-      bluePlate.density = 0
-      violetPlate.density = 1
+      bluePlate.density = 4
+      violetPlate.density = 3
       yellowPlate.density = 2
-      greenPlate.density = 3
-      pinkPlate.density = 4
+      greenPlate.density = 1
+      pinkPlate.density = 0
```

Comment:
Still many presets are broken. Note that you need to apply similar changes even to presets where densities were not specified, and the defaults densities equal to IDs were used. We can't use defaults anymore in those cases.